bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/whiteboard-toolbar-item/component.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '/imports/ui/components/button/component';
+import styles from '../styles';
+
+export default class WhiteboardToolbarItem extends Component {
+  constructor() {
+    super();
+
+    this._onClick = this._onClick.bind(this);
+  }
+
+  _onClick() {
+    const { objectToReturn, onItemClick } = this.props;
```

Comment:
Oswaldo, that was exactly the case that I had to avoid - since there is that rule that we discussed with you some time ago:
`JSX props should not use .bind()   react/jsx-no-bind`
Because in this case every time the component is recalculated - it gets binded again.
And as you might remember, the proper way for doing this would be either binding in the constructor (which is not applicable here, since we need to actually bind a variable), or [creating](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#lists-of-items)  another class, and passing down the variable that we need to return through `props`